Bottom: 4bae19b5d069e4e3c7923043c5b91d122a744fab
Top:    eae61c3bf8b09e14ff67ca2242f2794d291aa3c6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 15:39:05 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 79fdfe8..c234067 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -507,7 +507,7 @@ object3d_test_board_outline (void)
   object3d *board_outline;
 
   board_outline = object3d_from_board_outline ();
-  object3d_export_to_step (board_outline, "object3d_test.step");
+//  object3d_export_to_step (board_outline, "object3d_test.step");
   destroy_object3d (board_outline);
 }
